src/lib/Scenes/Artwork/Components/ArtworkTombstone.tsx

```diff
@@ -39,10 +39,13 @@ export class ArtworkTombstone extends React.Component<ArtworkTombstoneProps, Art
   renderSingleArtist(artist: Artist) {
     return (
       <React.Fragment>
-        <Flex flexDirection="row">
```

Comment:
Removes flex component and renders all the text in a text element for proper alignment.